developers/node-tutorial.md

```diff
@@ -1105,6 +1104,55 @@ celestia share get-by-namespace \
 
 ## Additional resources
 
+### Submitting a blob using curl
+
+In order to post a blob using curl, you will need a light node running
+with the `--core.ip string` flag, providing access to a consensus
+endpoint.
+The flag indicates node to connect to the given core consensus node.
+Examples: `127.0.0.1` or `subdomain.domain.tld`.
+Using either IP or DNS assumes RPC port 26657 and gRPC port 9090
+as default unless otherwise specified.
+
+1. In your terminal, set the auth token for the desired network. In this
+example, we will use Mainnet Beta.
+
+    ```bash
+    export CELESTIA_NODE_AUTH_TOKEN=$(celestia light auth admin --p2p.network celestia)
+    ```
+
+2. Post your blob with:
+
+    ```bash
```

Comment:
is there a reason the code snippet is indented vs the next section not having code snippet indented?

Reply:
it keeps the numbered lists from breaking, and also groups code blocks under their respective "step" ("1." "2." etc.)